Resources/config/util.xml

```diff
@@ -10,6 +10,8 @@
 
         <service id="fos_user.util.user_manipulator" class="FOS\UserBundle\Util\UserManipulator">
             <argument type="service" id="fos_user.user_manager" />
+            <argument type="service" id="event_dispatcher" />
+            <argument type="service" id="service_container" />
```

Comment:
I understand. Setter injection is not going to work either because the request service has been removed from Symfony 3.0.